packages/venia-concept/_buildpack/create.js

```diff
@@ -0,0 +1,166 @@
+        'clean',
+        'download-schema',
+        'lint',
+        'prettier',
+        'prettier:check',
+        'prettier:fix',
+        'start',
+        'start:debug',
+        'test',
+        'validate-queries',
+        'watch'
+    ];
+
+    const scriptsToInsert = {
+        build:
+            'yarn run clean && yarn run validate-queries && webpack --no-progress --env.mode production',
+        'build:dev':
+            'yarn run clean && webpack --progress --env.mode development'
+    };
+
+    return {
+        after({ options }) {
+            // The venia-concept directory doesn't have its own babel.config.js
+            // since that would interfere with monorepo configuration.
+            // Therefore there is nothing to copy, so we use the "after" event
+            // to write that file directly.
+            fs.outputFileSync(
+                resolve(options.directory, 'babel.config.js'),
+                "module.exports = { presets: ['@magento/peregrine'] };\n",
+                'utf8'
+            );
+        },
+        visitor: {
+            // Modify package.json with user details before copying it.
+            'package.json': ({
+                path,
+                targetPath,
+                options: { name, author }
+            }) => {
+                const pkgTpt = fs.readJsonSync(path);
+                const pkg = {
+                    name,
+                    private: true,
+                    version: '0.0.1',
+                    description:
+                        'A new project based on @magento/venia-concept',
+                    author,
+                    license: 'UNLICENSED',
+                    scripts: {}
+                };
+                toCopyFromPackageJson.forEach(prop => {
+                    pkg[prop] = pkgTpt[prop];
+                });
+
+                // The venia-concept template is part of the monorepo, which
+                // uses yarn for workspaces. But if the user wants to use
+                // npm, then the scripts which use `yarn` must change.
+                const toPackageScript = script => {
+                    const outputScript = script.replace(/\bvenia\b/g, name);
+                    return npmCli === 'npm'
+                        ? outputScript.replace(/yarn run/g, 'npm run')
+                        : outputScript;
+                };
+
+                scriptsToCopy.forEach(name => {
+                    pkg.scripts[name] = toPackageScript(pkgTpt.scripts[name]);
+                });
+                Object.keys(scriptsToInsert).forEach(name => {
+                    pkg.scripts[name] = toPackageScript(scriptsToInsert[name]);
+                });
+
+                if (!!process.env.DEBUG_PROJECT_CREATION) {
+                    setDebugDependencies(fs, pkg);
+                }
+
+                fs.outputJsonSync(targetPath, pkg, {
+                    spaces: 2
+                });
+            },
+            'package-lock.json.cached':
```

Comment:
This seems like a new set of files that we should all be aware of as I don't think they'll automatically update.

Can you please update the relevant docs or create an issue explaining what/how to update these files?

Reply:
Yes, we absolutely should.

Reply:
@sirugh Spurred by your sensible concerns, I've removed the lockfiles from the PR for now, and opened https://github.com/magento/pwa-studio/issues/1631 to add them back later.